cloud-platform/add-secrets-to-deployment.md

```diff
@@ -43,11 +43,22 @@ echo -n 'QUtJQUZUS1NBVzE1SEpMT0dE' | base64 -b0
 
 This will return the encoded secret 'UVV0SlFVWlVTMU5CVnpFMVNFcE1UMGRF' 
 
- 
+## Creating the secret
 
-put the encoded secret and id into a file (in this example secret-demo.k8s) as follows:
+There are 2 ways of creating the secret:
 
-```Yaml
+1. create a generic secret - for Concourse's git-crypt key
```

Comment:
Users aren't going to be using Concourse for now, so I don't think it's worth mentioning it at the moment.